spec/lib/services/swagger-api-service-spec.js

```diff
@@ -502,6 +502,17 @@ describe('Services.Http.Swagger', function() {
             });
         });
 
+        it('should throw 500 on invalid template', function() {
+            res.body = { message: "foo" };
+            return renderer(req, res, 'foo', mockNext)
+            .then(function() {
+                expect(mockNext).to.be.calledWithMatch({status: 500});
+            },
+            function(err) {
```

Comment:
I think you're right.  I was afraid that the assertion wouldn't fire if the promise were rejected, but returning a rejected promise should cause the test to fail.